app/utils/check-email.ts

```diff
@@ -2,9 +2,13 @@ import { InternalError } from '../middlewares/error-handler';
 import userService from '../services/users';
 import { conflictRequest, unprocessableEntity } from '../errors';
 
+export function checkEmailDomain(email: string): boolean {
+  const regexpEmail = /@wolox\.co$/;
+  return regexpEmail.test(email);
+}
+
 export async function checkEmail(email: string): Promise<InternalError | null> {
```

Comment:
I think we don't need these functions anymore. Let's remove all that's already being checked on the class-validator middleware